app/tasks.py

```diff
@@ -137,7 +137,12 @@ def import_product_db(db: Session, batch_size: int = 1000) -> None:
             execute_result = db.execute(
                 update(Product)
                 .where(Product.code == product_code)
-                .where(Product.source == Flavor.off)
+                .where(
+                    or_(
+                        Product.source == Flavor.off,
```

Comment:
maybe add a comment to explain why we have these 2 usecases ?

Reply:
Added